xipcode/test/sourcesScenario/packageOperationTest.js

```diff
@@ -33,17 +34,21 @@ describe('packageOperation', () => {
 				version: '2.2.2',
 				rootFolder: 'path/to/root',
 				buildReleaseFolder: 'build/release',
-				main: 'src/project1.js'
+				main: 'src/project1.js',
+				dependencies: ['dep1', 'dep2'],
+				nodeModulesFolder: 'path/to/node/modules/folder'
 			};
 		});
 
 		let sandbox;
 		let copyStub;
 		let writeJsonStub;
+		let npmStub;
```

Comment:
Consider calling this isXipcodeModuleStub.  Looks like the general pattern is to name stubs after the function name rather than the module.